### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

**Comment by dgleich** [comment]
@nassarhuda this looks fine to merge to me.

File: README.md [hunk 1]
```diff
@@ -5,16 +5,16 @@
 
 # MatrixNetworks
 This package consists of a collection of network algorithms.
-In short, the major difference between MatrixNetworks.jl and packages like LightGraphs.jl or Graphs.jl is the way graphs are treated.
+In short, the major difference between MatrixNetworks.jl and packages like [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl) or [Graphs.jl](https://github.com/JuliaGraphs/Graphs.jl) is the way graphs are treated.
 
-In [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl), graphs are created through Graph() and DiGraph() which are based on the representation of G as G = (V,E).
+In LightGraphs.jl, graphs are created through `Graph()` and `DiGraph()` which are based on the representation of $G$ as $G=(V,E)$.
```
Comment (nassarhuda) [inline README.md]: This should probably be `Graphs.jl` now?

**Review by nassarhuda — APPROVED** [review]
Thanks for the PR... sorry took me a while to look at it.

File: README.md (hunk 1 above)
Comment (pitmonticone) [reply README.md]: Yes, you're right. Changed. 

Do you believe the first sentence 

> In short, the major difference between MatrixNetworks.jl and packages like [LightGraphs.jl](https://github.com/JuliaGraphs/LightGraphs.jl) or [Graphs.jl](https://github.com/JuliaGraphs/Graphs.jl) is the way graphs are treated.

should be changed in 

> In short, the major difference between MatrixNetworks.jl and [Graphs.jl](https://github.com/JuliaGraphs/Graphs.jl) is the way graphs are treated.

**Comment by nassarhuda** [comment]
Awesome, thanks again for the PR.

**Pull request merged**